app/resources/base.html.template

```diff
@@ -109,7 +109,16 @@
   {% block body %}
     {% if env.amp and env.enable_analytics %}
       <amp-analytics config="https://www.googletagmanager.com/amp.json?id={{config.amp_analytics_id}}"
-                     data-credentials="include"></amp-analytics>
+                     data-credentials="include">
+        //AMP variable for dynamic GA property setting
+        <script type="application/json">
+        {
+          "vars": {
+          "gaProperty": "{{env.analytics_id}}"
```

Comment:
Indent this line by 2 more whitespaces.

Reply:
I think your suggested key name amp_gtm_id is better than config.amp_analytics_id. If possible can you rename this part too?

Reply:
Thanks gimite and githaruo!

@githaruo 
Can I just fix the line 111 or should I modify anywhere else to update the variable name?

Reply:
There are a few other references too. Try searching in the code repository:
https://github.com/google/personfinder/search?q=amp_analytics_id&unscoped_q=amp_analytics_id

Reply:
Looks the indent on **this line** (line 117) hasn't been fixed.

Reply:
done, thanks!

Reply:
Regarding the amp_analytics_id variable, let me raise another issue to fix them.